api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java

```diff
@@ -812,29 +832,32 @@ public void baseSetupWithStandardDataAndAuthentication() throws Exception {
 			Context.openSession();
 		}
 		
-		// The skipBaseSetup flag is controlled by the @SkipBaseSetup
-		// annotation. If it is deflagged or if the developer has
-		// marked this class as a non-inmemory database, skip these base steps.
-		if (useInMemoryDatabase()) {
-			if (!skipBaseSetup) {
-				if (!isBaseSetup) {
+		// The skipBaseSetup flag is controlled by the @SkipBaseSetup annotation. 		if (useInMemoryDatabase()) {
+		if (!skipBaseSetup) {
```

Comment:
What are these changes for?

Reply:
These are part of the original commits that i back ported as seen https://github.com/openmrs/openmrs-core/commit/d8ca28dc10f815137026755d71205620b51bf1a4